app/js/views/home.js

```diff
@@ -2,15 +2,34 @@ var app = app || {};
 
 app.views = app.views || {};
 
+var treesArray = [];
```

Comment:
`app.trees` will contain the tree data. It's not necessary to put it in the global namespace.